quickstarts/microsoft.mobilenetwork/mobilenetwork-provision-proxy-sims/prereqs/prereq.azuredeploy.parameters.json

```diff
@@ -0,0 +1,30 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "location": {
+            "value": "eastus"
+        },
+        "mobileNetworkName": {
+            "value": "GEN-UNIQUE"
+        },
+        "mobileCountryCode": {
+            "value": "001"
+        },
+        "mobileNetworkCode": {
+            "value": "01"
+        },
+        "sliceName": {
+            "value": "slice-1"
+        },
+        "dataNetworkName": {
+            "value": "internet"
+        },
+        "simGroupName": {
+            "value": "GEN-UNIQUE"
+        },
+        "encryptionKeyUrl": {
+            "value": "GEN-KEYVAULT-ENCRYPTION-KEY-URI"
+        }
+    }
+}
```

Comment:
Add newline to end of file.

Reply:
Not addressed to have the same code style across the templates.